src/coreclr/jit/emitxarch.cpp

```diff
@@ -1433,6 +1435,11 @@ bool emitter::TakesRex2Prefix(const instrDesc* id) const
 // - V'- bit to extend vvvv
 // - aaa - specifies mask register
 //    Rest    - reserved for future use and usage of them will uresult in Undefined instruction exception.
+// - u - Bit to indicate YMM Embedded rounding.
+//   Reserved for isas Avx10.1 and below
+//   Needs to be set to 0 for AVX10.2 adn above to indicate YMM embedded rounding
```

Comment:
nit:
```suggestion
//   Reserved for isas Avx10.1 and below
//   Set to 0 on AVX10.2 and above for YMM embedded rounding support
```

Or something along those lines. The current wording makes it sound like it must be always set to 0, rather than only conditionally set if we're using embedded rounding for YMM sizes

Reply:
I believe on Avx10.1 and below it is required to be set to `1` as well, so this should likely indicate that. Simply reserved doesn't imply a necessary state

Reply:
I have pushed these changes in my latest commit